3-bull-jobs/finished-app/data/dataHelper.js

```diff
@@ -1,8 +1,8 @@
-const _ = require('lodash');
+import _ from 'lodash';
 
-const redis = require('./redis');
+import { Redis } from './redis.js';
```

Comment:
redis from redis.js is already an instance of redis... naming convention suggest shouldn't start with capital...